packages/dtslint-runner/src/main.ts

```diff
@@ -62,7 +63,8 @@ export async function runDTSLint({
     inputs: testedPackages.map((path) => ({
       path,
       onlyTestTsNext: onlyTestTsNext || !packageNames.has(path),
```

Comment:
```suggestion
      onlyTestTsNext,
```

I feel like this should change too; it seems strange that we'd _enable_ testing of non-next versions when testing only next on the changed packages?

Reply:
Good catch -- that is pretty nonsensical.